common/include/common/common_utils.hpp

```diff
@@ -237,7 +237,7 @@ static inline std::vector<blas1_param_t> get_blas1_params(Args& args) {
   if (args.csv_param.empty()) {
     warning_no_csv();
     std::vector<blas1_param_t> blas1_default;
-    for (index_t size = 1048576; size <= 67108864; size *= 2) {
+    for (index_t size = 1048576; size <= 134217728; size *= 2) {
```

Comment:
Since we are working with large positive integer numbers that are power-of-two, we could represent them using the shift operator for increasing clarity, e.g., `1l<<20`.